src/Moryx.CommandCenter.Web/src/modules/components/ConsoleMethodConfigurator.tsx

```diff
@@ -15,60 +15,53 @@ export interface ConsoleMethodConfiguratorPropModel {
-                            <Col md={9}><span className="font-italic">{this.props.Method.description}</span></Col>
-                        </Row>
-                        <Row>
-                            <Col md={12} className="up-space-lg">
-                                { this.props.Method.parameters.subEntries.length === 0 ? (
-                                    <span className="font-italic">This method is parameterless.</span>
-                                ) : (
-                                    <NavigableConfigEditor Entries={this.props.Method.parameters.subEntries}
-                                                           ParentEntry={null}
-                                                           Root={this.props.Method.parameters}
-                                                           IsReadOnly={false}
-                                                           History={this.props.history}
-                                                           Location={this.props.location} />
-                                )}
-                            </Col>
-                        </Row>
-                        <Row className="up-space-lg">
-                            <Col md={12}>
-                                <Button color="primary"
-                                        className="float-right"
-                                        onClick={() => this.invokeSelectedMethod()}>
-                                    Invoke
-                                </Button>
-                            </Col>
-                        </Row>
-                    </Container>
-                )}
-            </div>
-        );
-    }
+    return (
+        <div>
+            {props.Method == null ? (
+                <span className="font-italic">Please select a method.</span>
+            ) : (
+                <Container fluid={true} className="no-padding">
+                    <Row>
+                        <Col md={3}><span className="font-bold">Name:</span></Col>
+                        <Col md={9}><span className="font-italic">{props.Method.displayName}</span></Col>
+                    </Row>
+                    <Row>
+                        <Col md={3}><span className="font-bold">Description:</span></Col>
+                        <Col md={9}><span className="font-italic">{props.Method.description}</span></Col>
+                    </Row>
+                    <Row>
+                        <Col md={12} className="up-space-lg">
+                            {props.Method.parameters.subEntries.length === 0 ? (
+                                <span className="font-italic">This method is parameterless.</span>
+                            ) : (
+                                <NavigableConfigEditor
+                                    Entries={props.Method.parameters.subEntries}
+                                    ParentEntry={null}
```

Comment:
Handing over the location and history is not necessary anymore?

Reply:
> [You'll also want to remove the history dependency from your package.json. The history library is a direct dependency of v6 (not a peer dep), so you won't ever import or use it directly. Instead, you'll use the useNavigate() hook for all navigation (see below).](https://reactrouter.com/en/main/upgrading/v5#upgrade-to-react-router-v6)

With the new hooks for navigating I don't see how we could use handing over the history or location. That way we can also remove the directly imported history library